### PR TITLE
Move tags to a sub menu flyout

### LINKED ISSUE #9585: Move tags to a sub menu flyout in the right click context menu so that users can see the tag name
The current ui for applying tags is a bit confusing and only shows the tag color, we can solve this issue by moving tags into a sub menu in the context menu flyout.

### PR DESCRIPTION
**Pull request by gave92**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #9585

**Details of Changes**
Add details of changes here.
- Moved tags to a standard sub menu flyout
- Moved MenuFlyoutItemWithImage to Menus folder
- Small refactor of MenuFlyoutHelper

**Validation**
How did you test these changes?
- [x] Built and ran the app

**Screenshots (optional)**

### CODE REVIEW

File: src/Files.Uwp/UserControls/Menus/ColoredToggleMenuFlyoutItem.xaml [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+                BorderThickness="{TemplateBinding BorderThickness}"
+                CornerRadius="{TemplateBinding CornerRadius}">
+                <Grid x:Name="AnimationRoot">
+
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto" />
+                        <ColumnDefinition Width="*" />
+                        <ColumnDefinition Width="Auto" />
+                    </Grid.ColumnDefinitions>
+                    <Border
```
Comment (yaichenbaum) [inline src/Files.Uwp/UserControls/Menus/ColoredToggleMenuFlyoutItem.xaml:33]: The round corners are a little blurred, can you try using an ellipse?

Comment (gave92) [reply src/Files.Uwp/UserControls/Menus/ColoredToggleMenuFlyoutItem.xaml]: Done, is it any better?

Comment (yaichenbaum) [reply src/Files.Uwp/UserControls/Menus/ColoredToggleMenuFlyoutItem.xaml]: Much better

**Review by yaichenbaum — DISMISSED** [review]
LGTM

**Comment by gave92** [comment]
image

**Comment by yaichenbaum** [comment]
> @yaichenbaum is this better?
> 
> * It does not need a custom control
> * Better readability of the checkmark
> 
> 

I liked the previous design, but I think this has better contrast.

**Review by yaichenbaum — APPROVED** [review]

**Pull request merged**